launch_scripts/eval_downstream.py

```diff
@@ -38,13 +42,20 @@ def main():
                         help="Load with FSDP, can be used to avoid OOMs")
     parser.add_argument("--max_new_tokens", type=int, default=None,
                         help="Override max new tokens, otherwise use task-specific default")
+    parser.add_argument("--is_hf_model", action="store_true",
+                        help="Use this flag if the model is a huggingface model")
     parser.add_argument("--include_image", action="store_true",
                         help="Include image in the evaluation outputs")
+    parser.add_argument("--save_dir", type=str, default=None,
```

Comment:
Is this needed? You can use `--save_dir` even without it and it will override the OmegaConf setting

Reply:
oh, I did not know that, we can remove it in that case